Sources/LokiLoggingProvider/PushClients/HttpPushClient.cs

```diff
@@ -32,7 +32,7 @@ public void Push(LokiLogEntry entry)
                     {
                         new[]
                         {
-                            $"{rfc3339Timestamp.Seconds}{rfc3339Timestamp.Nanos}",
+                            $"{rfc3339Timestamp.Seconds}{rfc3339Timestamp.Nanos.ToString("000000000")}",
```

Comment:
This is primarily for my reference so we know why it needs to be formatted to 9 dights.

https://protobuf.dev/reference/protobuf/google.protobuf/#timestamp

I'm also wondering if this would be better if you used the short hand `{rfc3339Timestamp.Nanos:000000000}` instead of `{rfc3339Timestamp.Nanos.ToString("000000000")}`.

Reply:
yes it would actually be better

Reply:
@SerIgel can you also fix the unit test in `HttpPushClientUnitTests.cs`? As soon as you do that I'll merge and publish a new package.

Thanks for your contribution.